### PR TITLE
Add volunteer tile to about page

### PR DESCRIPTION
**Pull request by diorshelton**
Adds Volunteer Tile into about page

### CODE REVIEW

File: src/components/Team.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    }
+  }
+`
+
+const Team = ({ eyebrowText, mainHeading, children, background }) => {
+  return (
+    <StyledTeam background={background}>
+      <div className="width-wrapper">
+        <div className="content">
+          <h2 className="heading-2">
```
Comment (daleinen7) [inline src/components/Team.jsx:49]: Let's change this to an H3

File: src/pages/about.jsx [hunk 2]
```diff
@@ -97,10 +101,25 @@ const AboutPage = () => {
           </EyeBrowHeaderSection>
           <ButtonGroup>
             <Button center={'center'} text={'Volunteer With Us'}></Button>
-            <Button center={'center'} text={'Partner With Us'}></Button>
+            <Button secondary center={'center'} text={'Partner With Us'}></Button>
           </ButtonGroup>
         </EyebrowHeaderGroup>
       </FAQSection>
+      <Team
```
Comment (daleinen7) [inline src/pages/about.jsx:108]: Please look up the "Heading" component and see if it's doing the same thing this component is doing. If it is, let's use that instead. 

However, if you have improved that component in any way, let's incorporate those improvements. 

(for example if it doesn't use React children, let's add it)

And take a look at the styles as well.

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Let's add spacing to the volunteer cards to match figma
Screen Shot 2023-07-10 at 7 21 54 PM

Otherwise, this looks great. Thanks!

File: src/pages/about.jsx (hunk 2 above)
Comment (daleinen7) [reply src/pages/about.jsx:108]: And so basically you'll want to cmd + shift + F and search for `import Heading` and there were 4 locations. You'll want to delete the link and text url props. Then replace the description prop with a wrapped `p` tag. 

But of course let me know if you have any follow up questions.

Comment (diorshelton) [reply src/pages/about.jsx]: I realized that adding a background prop to the Heading component only changed the background of the component and it has a max-width of 35...rem. I wasn't large enough to fit the about page. I added a Team wrapper and set the background on it. For some reason the CSS global variables aren't working (still looking into that).

**Comment by diorshelton** [comment]
Addressed changes covered in Standup yesterday. Looking forward to feedback.

**Comment by diorshelton** [comment]
> Let's add spacing to the volunteer cards to match figma Screen Shot 2023-07-10 at 7 21 54 PM
> 
> Otherwise, this looks great. Thanks!
Done.

**Review by daleinen7 — APPROVED** [review]
Looks good! Great work.

**Pull request merged**